backoffice/backoffice/workflows/tests/test_views.py

```diff
@@ -516,10 +516,27 @@ def test_search_data_name(self):
 
         response = self.api_client.get(url)
         results = response.json()["results"]
-
         assert len(results) == 1
         assert results[0]["data"]["name"]["value"] == "Smith, John"
 
+    def test_search_data_email(self):
```

Comment:
pytest has the feature called [parametrize](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html) for this kind of cases exactly

Reply:
I thought because it was a single test case this was not needed.

Django uses unittest in his classes which means `parametrize` is not supported: https://docs.pytest.org/en/latest/how-to/unittest.html#pytest-features-in-unittest-testcase-subclasses

Using [parametrized](https://pypi.org/project/parameterized/) to implement it